src/Illuminate/Foundation/Console/RouteListCommand.php

```diff
@@ -249,10 +249,7 @@ protected function isVendorRoute(Route $route)
      */
     protected function isFrameworkController(Route $route)
```

Comment:
We could probably drop this method and just use `$this->router->isFrameworkController((string) $route->getControllerClass())` directly, but I didn't want to introduce this potential breaking change since it's a `protected` method in a non final class